assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -80,6 +80,158 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	it( "should render a multi select question when the `question_type` is 'multi_select'", async () => {
+		registry
+			.dispatch( CORE_USER )
+			.receiveTriggerSurvey( fixtures.singleQuestionMultiSelect, {
+				triggerID: 'jestSurvey',
+			} );
+
+		fetchMock.post(
+			/^\/google-site-kit\/v1\/core\/user\/data\/survey-event/,
+			{ body: {}, status: 200 }
+		);
+
+		const { getByText, getByRole, getByLabelText, findByText } = render(
+			<CurrentSurvey />,
+			{
+				registry,
+			}
+		);
+
+		expect( fetchMock ).toHaveBeenCalledTimes( 1 );
+
+		// check correct question loads
+		expect(
+			getByText( 'What are your favorite pizza toppings?' )
+		).toBeInTheDocument();
+
+		// button should be disabled until two options are selected
+		expect( getByRole( 'button', { name: 'Next' } ) ).toHaveAttribute(
+			'disabled'
+		);
```

Comment:
The tests here are pretty comprehensive.  The only extra tests I would add (if we comprehensively test the MultiSelect in its own test suite) is variations on  `min_choices` and `max_choices` and different amounts of answers